### ISSUE TITLE
Can you provide the test set about Visa, ELPV, SDD, AITEX?

### ISSUE THREAD

**Issue by lalisa2020** [mala-lab/InCTRL#13]
I am concerned that the few normal images you provided may overlap with the test set I have personally split.

**Comment by Diana1026** [comment]
Hello, the normal samples are randomly selected from the training set of the target dataset. Hence, you can just use the whole test set and do not need a data split. Please refer to the content in Sec 4.1 of my paper. Thank you.

**Comment by lalisa2020** [comment]
Thanks for your reply!

However, I find that several datasets do not provide standard training and testing sets, requiring users to split them themselves. 

For example, the **original headCT dataset** consists of 100 normal slices and 100 abnormal slices, but **in Table 5**, your article uses the headCT dataset with 25 normal and 100 abnormal slices in the test set. The **original SDD dataset** contains 347 normal and 52 abnormal slices, but in the test set you used 286 normal and 54 abnormal slices.

<img width="383" alt="1713787199491" src="https://github.com/mala-lab/InCTRL/assets/155802306/9fd94b29-e4b8-4835-a8ea-3d6ac266ce95">

**Comment by Diana1026** [comment]
You should use [the convert scripts](https://github.com/mala-lab/InCTRL/tree/main/datasets/preprocess) to convert the datasets to MVTec AD format. Please follow the guidance in README.
